android-components/components/feature/media/src/main/java/mozilla/components/feature/media/middleware/RecordingDevicesMiddleware.kt

```diff
@@ -192,12 +193,15 @@ class RecordingDevicesMiddleware(
         private val processRecordingState: () -> Unit,
     ) : BroadcastReceiver() {
         override fun onReceive(context: Context, intent: Intent) {
-            ThreadUtils.postToMainThreadDelayed(
-                {
-                    processRecordingState.invoke()
-                },
-                NOTIFICATION_REMINDER_DELAY_MS,
-            )
+            val callingAppInfo = context.packageManager.getNameForUid(Binder.getCallingUid())
```

Comment:
This is for protecting against receiving the call from other apps, but maybe it's an overkill.

Reply:
No, I don't think so. I didn't see this comment until after I posted the above comment, so I guess I had the same concerns.